contracts/adapters/Configuration.sol

```diff
@@ -59,15 +60,30 @@ contract ConfigurationContract is
         bytes32 proposalId,
         bytes32[] calldata keys,
         uint256[] calldata values,
+        address[] calldata addresses,
         bytes calldata data
     ) external override onlyMember(dao) reentrancyGuard(dao) {
+        require(keys.length > 0, "missing keys");
         require(
-            keys.length == values.length,
-            "must be an equal number of config keys and values"
+            keys.length == values.length || keys.length == addresses.length,
+            "must be an equal number of keys and values/addresses"
         );
 
+        if (addresses.length == keys.length) {
```

Comment:
Here is to cover the case in which you are updating only the `address` config, in order to do that you need to make sure the addresses are valid so it is checking that during the proposal submission, otherwise it will fail only when the proposal is processed. Now that you mentioned we should be able to pass the `0` or `0x0` value to "delete" the config, I believe it makes sense to get rid of this statement.